templates/author.html

```diff
@@ -21,7 +21,7 @@
         <!--home-->
         <div class="card">
     <div class="card-body">
-      <h4 class="card-title">Salman Khan</h4>
+      <h4 class="card-title">{{ Tool.email_display_filter(author.mail) }}</h4>
       <p class="card-text">Hi, everyone, welcome to my personal page</p>
       <a href="#" class="card-link">Card link</a>
       <a href="#" class="card-link">Another link</a>
```

Comment:
Those two lines are useless for the time being, you may replace them with other useful options, or you could just remove them.

Reply:
Hi @Goldenbwuoy 

If you think line 26 and line 27 in author.html are useless, could you delete theme?

Thanks

**Hui**